kubernetes-services/work-scheduler/.eslintrc.yml

```diff
@@ -0,0 +1,9 @@
+rules:
```

Comment:
Can you add a short README at the top level of `kubernetes-services` to explain what goes under the directory and what one needs to do to define a new k8s service?

Reply:
I'll add the README, but not the template directory. The README will basically say "copy the work-scheduler" directory and modify it, so the work-scheduler directory IS the template.

Reply:
Done.